futures-executor/src/task_runner.rs

```diff
@@ -297,7 +297,8 @@ impl Future for SpawnedFuture {
     type Item = bool;
     type Error = bool;
 
-    fn poll(&mut self) -> Poll<bool, bool> {
-        self.inner.with_task_data(|f| f.poll())
+    fn poll(&mut self, ctx: &mut task::Context) -> Poll<bool, bool> {
```

Comment:
At least stylistically for this repo I'd personally prefer `cx` over `ctx`, but I may also be too used to rustc

Reply:
For what it is worth, `ctx` commonly stands for "context" in programming contexts (can be verified by google search "ctx programming" vs. "cx programming").

Reply:
+1 for ctx on the bikeshed, if for no other reason than that it will be obvious to folks coming from golang.

Reply:
It's probably not worth much, but I'm for `ctx` since I picked it in the first place and changing it would mean I'd have to change it. :)